target/consensource-ui
# Add featureFlag support for testing banner

Signed-off-by: Patrick-Erichsen <Patrick.Erichsen@target.com>

## Proposed change/fix

This PR adds support for configuring and using feature flags. The first use case of this feature is for a global footer to notify users that they are in a test instance.

## Types of changes

What types of changes does this pull request introduce to ConsenSource? _Put an `x` in the boxes that apply_


## How to run/test

- Open `.env.feature-flags` and set `ENABLE_TESTING_BANNER=true`
- `npm run watch`
- Visit http://localhost:8080/#!/ and confirm that there is a yellow test banner that reads: `This is a test instance of the ConsenSource application`
